build.gradle

```diff
@@ -254,54 +274,339 @@ allprojects {
+    implementation 'com.vladmihalcea:hibernate-types-52:2.16.3'
+    implementation group: 'commons-collections', name: 'commons-collections', version: '3.2.2'
+
+    testImplementation group: 'org.postgresql', name: 'postgresql', version: '42.5.0'
+    testImplementation group: 'org.testcontainers', name: 'postgresql', version: '1.17.5'
+    testImplementation "org.postgresql:postgresql:${postgresqlVersion}"
+    testImplementation "org.testcontainers:postgresql:1.17.2"
+    testImplementation "org.hamcrest:hamcrest-core:${hamcrestVersion}"
+    testImplementation "org.hamcrest:hamcrest-library:${hamcrestVersion}"
+    testImplementation "org.junit.jupiter:junit-jupiter-api:${junitJupiterVersion}"
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:${junitJupiterVersion}"
+    testRuntimeOnly "org.junit.vintage:junit-vintage-engine:${junitVintageVersion}"
+    testImplementation "org.junit.jupiter:junit-jupiter-params:${junitJupiterVersion}"
+    testImplementation('org.springframework.boot:spring-boot-starter-test')
+    testImplementation('org.springframework.cloud:spring-cloud-starter-contract-stub-runner:2.1.0.RELEASE') {
+      exclude group: 'com.github.tomakehurst', module: 'wiremock-standalone'
+      exclude group: 'commons-beanutils', module: 'commons-beanutils'
+    }
+    testImplementation ('com.github.tomakehurst:wiremock-jre8:2.25.1')
+    testImplementation ('org.springframework.cloud:spring-cloud-contract-wiremock:2.2.1.RELEASE') {
+      exclude group: 'commons-beanutils', module: 'commons-beanutils'
+    }
+    testImplementation 'com.github.npathai:hamcrest-optional:2.0.0'
+    testImplementation group: 'org.hamcrest', name: 'hamcrest-junit', version: '2.0.0.0'
+    testImplementation "io.github.openfeign:feign-jackson:${feignJackson}"
+    testImplementation 'io.github.openfeign.form:feign-form:3.8.0'
+    testImplementation  group: 'com.fasterxml.jackson.module', name: 'jackson-module-jaxb-annotations', version: '2.13.2'
+    testImplementation "org.exparity:hamcrest-date:2.0.4"
+    testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
+    testImplementation group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
+    testImplementation group: 'commons-lang', name: 'commons-lang', version: '2.6'
+    testImplementation(platform('org.junit:junit-bom:5.8.2'))
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+    testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', {
+      exclude group: 'junit', module: 'junit'
+      exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+    }
+    testImplementation 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.4.0'
+    testImplementation group: 'org.easymock', name: 'easymock', version: '5.0.1'
+    testImplementation group: 'org.jdom', name: 'jdom', version: '2.0.2'
```

Comment:
Should be org.jdom2, not org.jdom.  Original SDT used org.jdom2 version 2.0.4.

Reply:
added this dependency now: 
 testImplementation group: 'org.jdom', name: 'jdom2', version: '2.0.6.1'

Reply:
Confirmed dependency is now jdom2.